Emby.Server.Implementations/HttpServer/HttpListenerHost.cs

```diff
@@ -553,7 +557,7 @@ public async Task RequestHandler(IHttpRequest httpReq, string urlString, string
                     || ex is OperationCanceledException
                     || ex is SecurityException
                     || ex is FileNotFoundException;
-                await ErrorHandler(ex, httpReq, ignoreStackTrace).ConfigureAwait(false);
+                await ErrorHandler(ex, httpReq, ignoreStackTrace, urlToLog).ConfigureAwait(false);
```

Comment:
```suggestion
                await ErrorHandler(ex, httpReq, ignoreStackTrace, GetUrlToLog(urlString)).ConfigureAwait(false);
```

Then you can delete the definition of `urlToLog` in this method and also remove the conditional code in `ErrorHandler` since you will always have a non-empty URL to log

Reply:
But that would mean calling it twice, so I've moved `GetUrlToLog()` to `urlToLog` declaration